examples/gke/xfs-format-daemonset.yaml

```diff
@@ -0,0 +1,80 @@
+# Daemonset that converts ephemeral-storage to xfs
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: xfs-format
+spec:
+  selector:
+    matchLabels:
+      name: xfs-format
+  template:
+    metadata:
+      labels:
+        name: xfs-format
+    spec:
+      hostPID: true
+      nodeSelector:
+        cloud.google.com/gke-ephemeral-storage-local-ssd: "true"
+      tolerations:
+        - key: role
+          operator: Equal
+          value: scylla-clusters
+          effect: NoSchedule
+      initContainers:
+        - name: script-inject
+          image: scylladb/scylla-operator:latest
+          securityContext:
+            privileged: true
+          command:
+            - "/bin/bash"
+            - "-c"
+          args:
+            - |
+              set -euExo pipefail
+
+              cp -a /usr/local/lib/scylla-operator/gke/xfs-format /host/var/xfs-format
+              chmod a+x /host/var/xfs-format/xfs-format.sh
+              chmod a+x /host/var/xfs-format/bin/tune2fs
+
+              cp -a /usr/local/lib/scylla-operator/gke/systemd/. /host/etc/systemd/
+          volumeMounts:
+            - name: hostfs
+              mountPath: /host
+              mountPropagation: Bidirectional
+      containers:
+        - name: xfs-format
+          image: scylladb/scylla-operator:latest
+          securityContext:
+            privileged: true
+          command:
+            - "chroot"
+            - "/host"
+            - "/bin/bash"
+            - "-c"
+          args:
+            - |
+              set -euExo pipefail
+
+              systemctl daemon-reload
+              systemctl enable mnt-raid\\x2ddisks-disk0.mount
```

Comment:
`--now`? I guess ideally we need to wait for the mount to be actually created

Reply:
I don't think we care whether it's mounted before the reboot though?

Reply:
we'd at least fail sooner if something didn't work ?